docs/_diagrams/arch.uml

```diff
@@ -67,4 +65,20 @@ end note
 note top of Listener
   LISTEN session
 end note
+
+url of "PostgREST" [[https://github.com/PostgREST/postgrest/blob/main/main/Main.hs]]
+
+url of Auth is [[https://github.com/PostgREST/postgrest/blob/main/src/PostgREST/Auth.hs]]
+url of ApiRequest is [[https://github.com/PostgREST/postgrest/blob/main/src/PostgREST/ApiRequest.hs]]
+url of Plan is [[https://github.com/PostgREST/postgrest/blob/main/src/PostgREST/Plan.hs]]
+url of Query is [[https://github.com/PostgREST/postgrest/blob/main/src/PostgREST/Query.hs]]
+url of "Schema Cache" is [[https://github.com/PostgREST/postgrest/blob/main/src/PostgREST/SchemaCache.hs]]
+url of Admin is [[https://github.com/PostgREST/postgrest/blob/main/src/PostgREST/Admin.hs]]
+url of Config is [[https://github.com/PostgREST/postgrest/blob/main/src/PostgREST/Config.hs]]
+
+url of HTTPAPI is [[https://aosabook.org/en/posa/warp.html]]
+url of HTTPADMIN is [[https://aosabook.org/en/posa/warp.html]]
```

Comment:
This links to the Warp library, not sure if that's OK or if it should be removed.

Reply:
I think it's good :+1: